ownCloudSDK/XML/Parsing/OCXMLParser.m

```diff
@@ -243,8 +243,8 @@ - (void)parser:(NSXMLParser *)parser didEndElement:(NSString *)elementName names
 		}
 		else
 		{
-			// OCLogDebug(@"%@", lastParserElementOnStack);
-			// OCLogDebug(@"%@", [(OCXMLParserNode *)lastParserElementOnStack nodesForXPath:@"d:response/d:propstat"]);
+			OCLogDebug(@"%@", lastParserElementOnStack);
+			OCLogDebug(@"%@", [(OCXMLParserNode *)lastParserElementOnStack nodesForXPath:@"d:response/d:propstat"]);
```

Comment:
These two lines should stay commented out.

Reply:
Done 👍